math.md

```diff
@@ -183,7 +174,7 @@ Behold, a passing test.
 
 ```
 PASS
-ok  	github.com/gypsydave5/learn-go-with-tests/math/v1/clockface	0.006s
+ok  	clockface	0.006s
```

Comment:
This is an example of simplifying the passing tests output. I got confused at first by all the github.com/gypsydave5 references (as it was also in the imports for a few files), so I thought removing all the references might reduce confusion. However, I realize now that Go always outputs the full path so this simplification may suffer accuracy.